src/js/replaceelement.js

```diff
@@ -0,0 +1,9 @@
+const replaceElement = (id, inner) => {
+  const todoParentDiv = document.querySelector(`#${id}`);
```

Comment:
Do you need the string interpolation here?
I think you can use `id` directly, no? 🤔

Reply:
I had to use string interpolation as I needed to use `#` with `id` as this will let the API know its an `id` selector.